shiro/src/test/java/org/togglz/shiro/test/ShiroLoginServlet.java

```diff
@@ -19,7 +19,7 @@ public class ShiroLoginServlet extends HttpServlet {
     private static final long serialVersionUID = 1L;
 
     @Override
-    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
`doGet` method still exposes `ServletException`. I beleive it shouldn't be removed.
I've rechecked all three instances available in the project:
- jakarta.servlet:jakarta.servlet-api:6.0.0
- jakarta.platform:jakarta.jakartaee-web-api:10.0.0
- org.apache.tomcat.embed:tomcat-embed-core:10.1.4

Reply:
not sure if I get it. I would say it's not needed as a checked exception here anymore. If it would, then the compiler should complain about it when it's missing. Maybe they changed it from checked to unchecked exception?

I would not add the ServletException here again. I think there is no benefit of doing it.